package.json

```diff
@@ -8,10 +8,10 @@
     "build": "yarn compile-css",
     "compile-css": "node-sass src/components/index.scss src/components/design-system.css && yarn format",
     "cy:open": "cypress open",
-    "cy:run": "cypress run --browser chrome --record",
+    "cy:run": "cypress run --browser chrome --config video=false",
     "develop": "chokidar src/**/*.scss -c 'yarn build' --initial",
-    "e2e:ci": "CYPRESS_SUPPORT=y yarn run build && start-server-and-test styleguide http://localhost:9008 cy:open",
-    "e2e:dev": "CYPRESS_SUPPORT=y start-server-and-test styleguide http://localhost:9008 cy:open",
+    "e2e:ci": "export NODE_ENV=ci && start-server-and-test styleguide http://localhost:9008 cy:run",
```

Comment:
Travis supports Env variables in their dashboard, so there is no need to add `NODE_ENV=ci`

Reply:
I've actually moved this over into the yml config which feels correct.

Reply:
That works as well. If you ever wanted to put sensitive env variables, the way to put it is to go to settings -> environment variables
<img width="969" alt="Screen Shot 2019-10-28 at 8 49 48 PM" src="https://user-images.githubusercontent.com/11466401/67736439-e73dbd00-f9c4-11e9-8a7a-183b51c4debc.png">

Reply:
Well, I understand that you can actually [encrypt those too](https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml) and still put in the `env` stanza, but we just needed a boolean to indicate CI.

Reply:
Wow amazing. TIL